app/models/catch.rb

```diff
@@ -0,0 +1,12 @@
+class Catch < ApplicationRecord
+  belongs_to :user
+
+  validates :species, presence: true
+  validates :weight, presence: true
+  validates :length, presence: true
+  validates :spot_name, presence: true
+  validates :latitude, presence: true
+  validates :longitude, presence: true
+  validates :lure, presence: true
+  validates :photo_url, presence: true
```

Comment:
Do we want to force a user to upload a photo? Or should we forego this validation?

Reply:
good call taking care of that now